python/nav/web/__init__.py

```diff
@@ -38,6 +41,7 @@
 def refresh_session(request):
     """Forces a refresh of the session by setting the modified flag"""
     request.session.modified = True
+    _logger.debug('Session refreshed in "refresh_session"')
```

Comment:
Not entirely convinced this is needed, as the access log will show this view being accessed regularly...

Reply:
I tend to eye-grep for the DEBUG-lines only when debugging with logs :)